src/db/filename.rs

```diff
@@ -130,10 +130,22 @@ mod tests {
     use super::*;
 
     #[test]
-    #[cfg(unix)]
     fn test_generate_filename() {
         let dirname = "test";
-        let mut tests = vec![
+        let mut tests = if cfg!(windows) {
+        let tests_windows =  vec![
```

Comment:
I think maybe you could remove the redundant assignment to `tests_windows` and `tests_unix` like below?
```rust
let mut tests = if cfg!(windows) {
    vec![/* something */]
} else {
    vec![/* something */]
}
```

Reply:
ok，I will fix it